Upstream: https://sources.debian.net/src/pal/0.4.3-8.1/debian/patches/53_debian_rl_hook.patch/

Description: FTBFS fix, replace Function with int (*)(void)
Author: Maximiliano Curia <maxy@debian.org>
Bug: http://bugs.debian.org/741779
Forwarded: no
Last-Update: 2014-04-25
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: src/manage.c
===================================================================
--- src/manage.c	2014-04-25 19:56:03.242689939 +0200
+++ src/manage.c	2014-04-25 20:03:50.235382801 +0200
@@ -403,7 +403,7 @@
     rl_initialize();     /* Initialise readline so we can fiddle stuff */
     rl_already_prompted = 1;
     rl_redisplay_function = pal_rl_ncurses_hack;
-    rl_pre_input_hook = (Function*) pal_rl_ncurses_hack;
+    rl_pre_input_hook = pal_rl_ncurses_ihack;
 
 
     /* initialize curses */
Index: src/rl.c
===================================================================
--- src/rl.c	2014-04-25 20:02:59.173999681 +0200
+++ src/rl.c	2014-04-25 20:03:17.606499242 +0200
@@ -209,6 +209,12 @@
     refresh();
 }
 
+int pal_rl_ncurses_ihack(void)
+{
+    pal_rl_ncurses_hack;
+    return 0;
+}
+
 gboolean pal_rl_get_y_n(const char* prompt)
 {
     gchar *s = NULL;
Index: src/rl.h
===================================================================
--- src/rl.h	2014-04-25 20:02:51.901802596 +0200
+++ src/rl.h	2014-04-25 20:03:12.590363056 +0200
@@ -41,4 +41,5 @@
 void pal_rl_completions_output(char **matches, int num_matches, int max_length );
 PalEvent* pal_rl_get_event(GDate** d, gboolean allow_global);
 void pal_rl_ncurses_hack(void);
+int pal_rl_ncurses_ihack(void);
 #endif
